src/test/java/com/vmware/vip/i18n/TranslationMessageTest.java

```diff
@@ -258,26 +258,23 @@ public void testGetComponentMessages_() {
     }
 
     @Test
-    public void testGetAllComponentTranslation() {
+    public void testGetAllComponentTranslationOnlineModeOnly() {
         List<Map> list = new ProductService(dto).getAllComponentTranslation();
-        Assert.assertTrue(list.size() > 0);
+        Assert.assertTrue(list.size() == 24); // 8 locales
     }
 
     @Test
     public void testGetAllComponentTranslationMixedMode() {
         String offlineResourcesBaseUrlOrig = vipCfg.getOfflineResourcesBaseUrl();
         vipCfg.setOfflineResourcesBaseUrl("offlineBundles/");
         List<DataSourceEnum> msgOriginsQueueOrig = vipCfg.getMsgOriginsQueue();
-        vipCfg.setMsgOriginsQueue(new LinkedList<DataSourceEnum>(Arrays.asList(DataSourceEnum.VIP, DataSourceEnum.Bundle)));
-        String vipServerOrig = vipCfg.getVipServer();
-        vipCfg.setVipServer("http://1.1.1.1:80");
```

Comment:
Why remove this line?

Reply:
Because we want the server hostname to be correct to enable online mode (the one that has a mocked response).